### PR TITLE
rhcos-toolbox: Add help switch

### PR DESCRIPTION
**Pull request by ashcrow**
See https://bugzilla.redhat.com/show_bug.cgi?id=1684258

### CODE REVIEW

**Comment by ashcrow** [comment]
Removed "command" since it would interfere with passing debug commands to the toolbox container.

File: rhcos-toolbox [hunk 1]
```diff
@@ -122,8 +122,32 @@ container_exec() {
...
+
+You may override the following variables by setting them in ${TOOLBOXRC}:
+- REGISTRY: The registry to pull from. Default: $REGISTRY
+- IMAGE: The image and tag from the registry to pull. Default: $IMAGE
+- TOOLBOX_NAME: The name to use for the local container. Default: $TOOLBOX_NAME
+
+Example toolboxrc:
+REGISTRY=my.special.registry.example.com
+IMAGE=debug:latest
+TOOLBOX_NAME:special-debug-container"
```
Comment (yuqi-zhang) [inline rhcos-toolbox]: The `:` should be a `=`

Comment (ashcrow) [reply rhcos-toolbox]: oops! Fixing

**Comment by ashcrow** [comment]
@yuqi-zhang updated based off review.

**Comment by ashcrow** [comment]
This is ready to merge if it passes the double :+1: criteria.

**Pull request merged**